tools/fixFormat.js

```diff
@@ -67,6 +67,21 @@ async.each(packages, function(item, callback) {
     pkg.licenses = pkg.license;
     delete pkg.license;
   }
+  if (pkg.npmFileMap) {
+    for (var i in pkg.npmFileMap) {
+      var npmbasePath = pkg.npmFileMap[i].basePath;
+      if (npmbasePath && npmbasePath.length != 0 && ((typeof npmbasePath) == "string")) {
+          npmbasePath = (npmbasePath).replace(/^\/|\/$/g , "");
+          pkg.npmFileMap[i].basePath = npmbasePath;
+      }
+    }
+  } else if (pkg.autoupdate) {
```

Comment:
Let's just use `if`, not `else if`, you don't know which one the user want to preserve if there are multi configs.